README.md

```diff
@@ -9,7 +9,7 @@
 Docker Community Edition (CE) >= 17.06
 Docker Compose >= 1.18
 
-Follow the [Docker installation instructions](https://www.docker.com/products/overview) for your OS.
+Follow the [Docker installation instructions](https://www.docker.com/get-docker#/overview) for your OS.
```

Comment:
Thanks for the suggestion, but maybe this link is even more direct: https://www.docker.com/community-edition#/download

It seems that Docker is trying to push their Enterprise Edition product more, so it's a bit more difficult to navigate to the free Community Edition.

Reply:
okiedoke